ci/jenkins/jobs/job-templates.yaml

```diff
@@ -115,6 +115,12 @@
         artifact-num-to-keep: -1
         days-to-keep: 7
         num-to-keep: 30
+    - throttle:
+        max-per-node: 1
```

Comment:
is that different from `concurrent: false`?

Reply:
Yes, "concurrent" is for the same job. "throttle" covers several jobs with category value.